nlp_apps.ipynb

```diff
@@ -572,7 +559,7 @@
    "source": [
     "disputed = list(range(49, 58+1)) + [18, 19, 20, 64]\n",
     "jay, madison, hamilton = [], [], []\n",
-    "for i, p in enumerate(papers):\n",
+    "for i, p in enumerate(stemmed_papers):\n",
```

Comment:
You have not defined `stemmed_papers`.

As I previously suggested, it would be better if you added the stemming section after the original code, to compare the two.